### ISSUE TITLE
Validation set and test set issues

### ISSUE THREAD

**Issue by libaitian-oss** [mkang315/CST-YOLO#7]
Hi author, I would like to ask if your final result is the result obtained under the validation set? Also, are the results from each round of training under the train path or the val path?So what should I do if I want to use the pictures from the test set to get the final result?
![图片1(1)](https://github.com/user-attachments/assets/6a618cb2-5649-4643-8ee1-e963a3f0480b)

**Comment by mkang315** [comment]
Please set the test set path of the dataset in the .yaml configure file. Take [bccd.yaml](https://github.com/mkang315/CST-YOLO/blob/main/data/bccd.yaml) as an example, I comment the original path of the validation set and add the test set as follows:
```
train: ./datasets/bccd/traindata
val: ./datasets/bccd/testdata
# val: ./datasets/bccd/validata
```
